powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/model/TokenInfo.java

```diff
@@ -38,12 +37,10 @@ public TokenInfo() {
      * Parameterized constructor.
      * @param tokenId Token identifier.
      * @param tokenSecret Token secret.
-     * @param expires Expiration timestamp as Unix timestamp in milliseconds.
      */
-    public TokenInfo(String tokenId, String tokenSecret, Long expires) {
+    public TokenInfo(String tokenId, String tokenSecret) {
```

Comment:
What happened here? Is this something we should document?

Reply:
See: https://github.com/wultra/powerauth-server/issues/447